fe/lib/components/QuestionnaireBuilder/components/GraphqlQueryRenderer/GraphqlQueryRenderer.tsx

```diff
@@ -59,7 +59,7 @@ export const convertComponentsToMutationVariables = (
   hirerId: string,
 ): MutationVariables => {
   const questionnaireComponents: MutationQuestionnaireComponents[] = components.map(
-    (component) =>
+    component =>
```

Comment:
Oh yeah Prettier does not get along well with this repo at the moment 😢

```suggestion
    (component) =>
```